modules/covid_vaccine/app/services/covid_vaccine/v0/vetext_configuration.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'common/client/configuration/rest'
+
+module CovidVaccine
+  module V0
+    class VetextConfiguration < Common::Client::Configuration::REST
+      self.read_timeout = Settings.vetext.timeout || 15
+
+      def base_path
+        Settings.vetext.url
+      end
+
+      def service_name
+        'Vetext'
+      end
+
+      def connection
+        # TODO: Remove SSL no-verify option!
+        Faraday.new(base_path, headers: base_request_headers, request: request_options, ssl: { verify: false }) do |c|
```

Comment:
We need to take out this `ssl: { verify: false }` option, waiting for current test run before I do that.